.circleci/config.yml

```diff
@@ -12,9 +12,8 @@ jobs:
           password: $DASH_PAT_DOCKERHUB
     steps:
       - checkout
-      - browser-tools/install-browser-tools:
-          chrome-version: 116.0.5845.96
-          firefox-version: 116.0.3
+      - browser-tools/install-chrome
```

Comment:
These diffs are all coming from #281 right? So we'll need to merge #281 first, rebase, then merge in these changes..?

Reply:
Yes. That is correct.